@kiva/kv-components/vue/KvSideSheet.vue

```diff
@@ -190,6 +199,7 @@ export default {
 			closeSideSheet,
 			goToLink,
 			modalStyles,
+			avoidBodyScroll,
```

Comment:
Do we need to export this method now that it's just being used in the component?